assignments/number-functions/code.js

```diff
@@ -0,0 +1,10 @@
+const add=(a,b) => a+b
+const subtract=(a,b) =>a-b
+const multiply=(a,b) =>a*b
+const divide=(a,b) =>a/b
+const averageOfTwo=(a,b) =>(a+b)/2
+const averageOfThree=(a,b,c) =>(a+b+c)/3
+const distance=(a,b) => Math.abs(a-b)
+const manhattanDistance=(x1,x2,y1,y2)=> Math.abs(x1-y1)+ Math.abs(x2-y2)
```

Comment:
This is a bit odd since the names of the arguments don't match the way they're being used. The idea of Manhattan distance is you measuring the distance along the x axis (i.e. `Math.abs(x1 - x2)`) and the y axis and adding them together. And this function is intended to be called with the first two arguments being one x,y point and the third and fourth being another. But since you've named the variables as if it was two x coordinates and then two y coordinates, then you changed the code to make it work. But it doesn't make sense to a human reader until they understand that the names don't match what the values really represent.

Also note that you could make this even more clear by using the `distance` function you just defined:

```suggestion
const manhattanDistance=(x1, y1, x2, y2)=> distance(x1, x2)+ distance(y1, y2);
```

This makes it *really* clear what's going on: the Manhattan distance is the sum of two distances, one on the x-axis and one on the y-axis.